### ISSUE TITLE
Code block escaping: the return

### ISSUE THREAD

**Issue by tlienart** [tlienart/Franklin.jl#513]
As indicated by @wikunia  

`````
```
{{code.blah}}
```
`````

will not do the expected and try to  fill a hfun (and fail so it will show nothing). The issue is that the `{{` gets properly left alone in the markdown pass (bc it's a code block) but then appears as part of a `<pre>` block in the HTML where it gets picked up in the second round. 

Two ways around this:

1. (easy) ignore `{{...}}` blocks that are in `<pre>`, same as the `<script>` one
1. (harder) escape code blocks in general

the latter is the more natural but is also more work to check that it doesn't fail and produce horrible results.

I might just do the first one for now and revisit the code escaping rules in full details in some future

**Comment by Wikunia** [comment]
I think this is the same category:
`````
```
<script src="https://code.jquery.com/jquery-3.3.1.js"></script>
<script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>
```
`````
I know I should only display julia code :smile: but yeah I think nothing should be parsed in code blocks.

**Comment by tlienart** [comment]
sorry what's the issue in your example?

**Comment by Wikunia** [comment]
It doesn't show the code but instead adds the script.

I also had
`````
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css">
```
`````

which destroyed the design of my blog

**Comment by tlienart** [comment]
yes in both cases adding `html` to  the fence  should fix the issue (could you confirm?)

PS: I agree that this is part of a broader issue of systematically escaping code blocks.

**Comment by Wikunia** [comment]
Oh yeah that works. Thanks! I tried `js` instead of `html` which of course is wrong...

**Comment by Wikunia** [comment]
Another example

```cpp
#include <iostream>
```

only shows `#include`

**Comment by tlienart** [comment]
Yes that one had also been reported :)

**Comment by tlienart** [comment]
Ps maybe more useful is that you can write something that does the escaping for you, see the config.md in franklindocs.

Of course this needs a native solution but in the mean time the above will help

**Comment by Wikunia** [comment]
Just encountered that my approach with using `&lt;, &gt; &123;` does not work when using prerendering. And using `\esc` with prerendering produces `\n` instead of actual multi-line output.

**Comment by tlienart** [comment]
Closed now by #610
